VATSpy.dat

```diff
@@ -17752,7 +17706,7 @@ ZWAK|Aksu Wensu|41.263056|80.289444||ZWUQ|0
 ZWAT|Aletai|47.750278|88.084444||ZWUQ|0
 ZWBL|Bole Alashankou|44.895|82.300556||ZWUQ|0
 ZWCM|Qiemo Yudu|38.233611|85.464444||ZWUQ|0
-ZWFY|Fuyun Keketuohai|46.804167|89.512222||ZWUQ|0
+ZWFY|Fuyun Keketuohai|46.804167|89.512222||ZWUQ|0	

```

Comment:
Some trailing whitespace seems to have accidentally been added here

Reply:
Wops, resolved.